### DISCUSSION TITLE
Control `type: checkboxes` fields in issue forms via URL params

### DISCUSSION THREAD

**Discussion by leonhma** [community/community#32200] [Projects and Issues]
I am currently using issue forms in combination with query parameters in order to prefill some fields (you can use the ids of the fields as keys) to link users from my application directly to a prefilled bug report with error logs, latest-version-check, ... The only thing that I am not able to control is the `type: checkboxes` field. There is also no documentation on this. I know issue forms is in beta, but if this works with regular fields it has to work with checkboxes too, right? Does anyone know how I set the state of the checkboxes via a query param? I have tried `checkboxes=(true,true)`, `[true, true]`, `true, true`, `[1,1]` and it didn't work so far.

**Comment by ofek** [comment]
Is this somehow possible today? I couldn't find a way.
